packages/diagrams/lib/src/floating_action_button.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+
+import 'diagram_step.dart';
+
+const String _basic = 'floating_action_button';
+const String _labeled = 'floating_action_button_label';
+
+class FloatingActionButtonDiagram extends StatelessWidget implements DiagramMetadata {
+  const FloatingActionButtonDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+    Widget returnWidget;
+
+    switch (name) {
+      case _basic:
+        returnWidget = Scaffold(
+          appBar: AppBar(
+            title: const Text('Floating Action Button Sample'),
```

Comment:
```suggestion
            title: const Text('Floating Action Button'),
```
Looks like the 'Sample` was being truncated in the diagram. We can just remove it so there isn't an ellipsis in the heading.